site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
Doesn't this allow users to be created with the same name as an organization, and the other way around? Right now this case will be caught (rather by luck) in `UserOrgNameValidator::userOrOrgReserved()`, but let's not depend on that. I'm proposing to remove both `CHECK_*` constants and unconditionally do the check in `userOrOrgNameExists()`. It's not in a hot path anyways, so the overhead of double-checking should be acceptable.